tools/sidekick/library.json

```diff
@@ -0,0 +1,13 @@
+{
+  "project": "adobe-io",
+  "host": "https://www.hlx.live/docs/sidekick#customizing-the-sidekick",
```

Comment:
You had it right before.. This needs to be your production hostname i.e developer.adobe.com?